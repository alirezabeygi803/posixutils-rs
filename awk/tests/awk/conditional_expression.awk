BEGIN {
	print 1 ? "t" : "f"
    print 0 ? "t" : "f"
	print 1 ? 1 ? "t" : "f1" : "f2"
	print 0 ? 1 ? "t" : "f1" : "f2"
	print 1 ? 0 ? "t" : "f1" : "f2"
}