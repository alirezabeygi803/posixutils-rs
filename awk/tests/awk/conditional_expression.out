t
f
t
f2
f1
